.github/workflows/storybook.yml

```diff
@@ -0,0 +1,13 @@
+name: Storybook
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  build-storybook:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Deps
+        run: npm install
+      - name: Build Storybook
+        run: npm run build-storybook
```

Comment:
would love to go one step further at some point and add a jest test for viewing the different stories